packages/prosemirror-lwdita-backend/src/api/modules/octokit.module.mts

```diff
@@ -18,13 +18,21 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 import { Octokit } from "@octokit/rest";
 import { createOAuthAppAuth } from "@octokit/auth-oauth-app";
 import { Endpoints } from "@octokit/types";
-import dotenv from 'dotenv'; 
+import dotenv from 'dotenv';
 import { retry } from "@octokit/plugin-retry";
-dotenv.config();  // Load environment variables from .env file 
+import * as fs from 'fs';
+
+dotenv.config();  // Load environment variables from .env file
+
+/**
+ * Load the configuration
+ */
+const config = JSON.parse(fs.readFileSync('./server-config.json', 'utf8'));
```

Comment:
@plutonik-a Typescript allows us to import json directly without having to use the fs module
So please change this to 
```ts
import config from './server-config.json`
```
And add this in the top level tsconfig
```json
    "resolveJsonModule": true,                /* Allows importing JSON files */
```

Reply:
Due to import issues in this specific package it seemed to be the only viable option.